projects/nuvoton/numaker_iot_m487_wifi/uvision/aws_demos/aws_demos.uvproj

```diff
@@ -374,7 +374,7 @@
 							<MiscControls>--diag_suppress=550,177,C4017,111,2770,223</MiscControls>
```

Comment:
[projects/nuvoton/numaker_iot_m487_wifi/uvision/**aws_demos_eth**/aws_demos.uvproj](https://github.com/aws/amazon-freertos/blob/master/projects/nuvoton/numaker_iot_m487_wifi/uvision/aws_demos_eth/aws_demos_eth.uvproj)
Will also need to be updated manually.